drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/importer/importer.rb

```diff
@@ -169,13 +169,19 @@ def ingest!
       # as pending deletion.  We'll remove the pending where appropriate
       mark_tree_as_dead(Date.current)
 
+      # Add Export row
+      add_export_row
+
       # Add any records we don't have
       add_new_data
 
       # Process existing records,
       # determine which records have changed and are newer
       process_existing
 
+      # Update all ExportIDs
```

Comment:
Something like 'Update all ExportIDs for corresponding existing warehouse records'?